SUMMARY.md

```diff
@@ -6,6 +6,20 @@
     * [Data Sources](data-sources.md)
     * [Software](software.md)
 * [User Manual](user-manual.md)
+    * [Data Products](data-products.md)
+        * [Infrared head imaging](flir.md)
+        * [Template](template.md)
+        * Photochemical reflectance index
+        * [Fluorescence intensity imaging](fluorescence-intensity-imaging.md)
+        * Short-wave infrared
```

Comment:
SWIR and VNIR could be combined into a single file, or at least they should be next to each other.

If we are describing the data products, this could be a single page called 'hyperspectral imaging data' thoughts @czender ?